app/Converters/parser_onnx.py

```diff
@@ -5,15 +5,17 @@
 from onnx import helper, numpy_helper
 from ultralytics import YOLO
 
+
 def convert_pt_to_onnx(pt_model_path, onnx_model_path=None):
     if onnx_model_path is None:
         onnx_model_path = pt_model_path.replace('.pt', '.onnx')
 
     model = YOLO(pt_model_path)
-    model.export(format="onnx", dynamic=False, simplify=True)
+    model.export(format="onnx", dynamic=False, simplify=False)
```

Comment:
Why `simplify=False`? If simplification caused issues, add a comment or a flag to control it; otherwise prefer `simplify=True` for a smaller, stable graph.